src/style.css

```diff
@@ -0,0 +1,89 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+  font-style: normal;
+}
+
+html {
+  font-family: Avenir, sans-serif;
+}
+
+.search_form {
+  margin-top: 20px;
+
+}
+
+.search_icon {
+  position: absolute;
+}
+
+.search_icon--big {
+  height: 19px;
+  width: 19px;
+  padding-left: 26px;
+  padding-top: 25px;
+
+}
+
+.search_icon--small {
```

Comment:
check this checklist point
<img width="832" alt="image" src="https://user-images.githubusercontent.com/102411130/221127357-dad0a6cf-0646-4835-bae6-3b39ff9c2403.png">

Reply:
![image](https://user-images.githubusercontent.com/108753306/221146312-56bcbf23-935d-4d25-bc83-0f220d462f84.png)
![image](https://user-images.githubusercontent.com/108753306/221146427-9c1fe238-e920-449c-a177-7fc60fa96462.png)
Please need help!!!
When I run tests through npm, everything is okay, but when the tests are run on GitHub, it gives an error of size difference in small search bar

Reply:
Problem solved! tnx